src/main/java/com/talentpath/airTNB/models/Listing.java

```diff
@@ -20,16 +20,11 @@
 
     private String subTitle;
 
-    @Digits(integer=3, fraction=6)
-    private BigDecimal longitude;
-
-    @Digits(integer=3, fraction=6)
-    private BigDecimal latitude;
-
     private String state;
```

Comment:
City and State should be tied to location, would like to talk through the logic of separating them